.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 test/fixtures/*.actual.css
+.idea/
```

Comment:
It's like .DS_Store & Thumbs.db.
You should ignore editor prefs in your global gitignore. I don't want to see everyone editor files here. I think you will thanks me later.